tests/acceptance/test_permissions_aragon.py

```diff
@@ -0,0 +1,226 @@
+from brownie.network.state import TxHistory
+from utils.config import contracts
+from tests.regression.test_permissions import protocol_permissions
+from utils.config_mainnet import (
+    lido_dao_agent_address, lido_easytrack_evmscriptexecutor, lido_easytrack_evmscriptexecutor)
+
+
+def has_permissions(app, role, entity):
+    return contracts.acl.hasPermission(entity, app, role)
+
+
+def assert_has_permissions(app, role, entity):
+    assert True == has_permissions(
+        app, role, entity), f'Entity {entity} should have permission {role} for app {app}'
+
+
+def assert_has_not_permissions(app, role, entity, role_name=None):
+    assert False == has_permissions(
+        app, role, entity), f'Entity {entity} should not have permission {role_name or role} for app {app}'
+
+
+def assert_has_not_permissions_in_list(app, role, entity_list, role_name=None):
+    for address in entity_list:
+        assert_has_not_permissions(app, role, address, role_name)
+
+
+def assert_has_permissions_in_list(app, role, entity_list):
+    for address in entity_list:
+        assert_has_permissions(app, role, address)
+
+
+def assert_not_match_with_events_in_list(app, role, entity_list, role_name=None, app_name=None):
+    for address in entity_list:
+        assert_not_match_with_events(app, role, address, role_name, app_name)
+
+
+def assert_not_match_with_events(app, role, entity, role_name=None, app_name=None):
+    assert not has_permissions(app, role, entity)
+
+
+def collect_permissions_from_events(permission_events):
+    apps = {}
+    for event in permission_events:
+        if event['allowed'] == True:
+            if event['app'] not in apps:
+                apps[event['app']] = {str(event['role']): [event['entity']]}
+            elif event['app'] in apps:
+                if str(event['role']) not in apps[event['app']]:
+                    apps[event['app']][str(event['role'])] = [event['entity']]
+                else:
+                    apps[event['app']][str(event['role'])].append(
+                        event['entity'])
+
+    return apps
+
+
+def permissions_after_votes():
+    return {
+        contracts.acl.address: {
+            'roles': {
+                'CREATE_PERMISSIONS_ROLE': [contracts.voting.address]
+            }
+        },
+        contracts.kernel.address: {
+            'roles': {
+                'APP_MANAGER_ROLE': [contracts.voting.address]
+            }
+        },
+        '0x853cc0D5917f49B57B8e9F89e491F5E18919093A': {
+            'roles': {
+                'REGISTRY_MANAGER_ROLE': [contracts.voting.address],
+                'REGISTRY_ADD_EXECUTOR_ROLE': [contracts.voting.address]
+            }
+        },
+        contracts.token_manager.address: {
+            'roles': {
+                'ASSIGN_ROLE': [contracts.voting.address]
+            }
+        },
+        contracts.lido.address: {
+            'roles': {
+                'PAUSE_ROLE': [contracts.voting.address],
```

Comment:
would suggest checking that neither `BURN_ROLE` nor `DEPOSIT_ROLE` is assigned after the vote
![image](https://user-images.githubusercontent.com/961317/234672947-af616c2f-e5ce-48f3-a806-6dac1b8492c4.png)

cc: @arwer13 @bulbozaur 

Though, decided today not to invest a significant amount of time for the sake of ACL perfection.